File: SoftUni - C# OOP/Interfaces and Abstraction/Interfaces and Abstraction - Exercise/05. Birthday Celebrations/Citizen.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _05._Birthday_Celebrations
{
    public class Citizen : ICitizen
    {
        public Citizen(string name,int age,string id,string birthdate)
        {
            this.Name = name;
            this.Age = age;
            this.Id = id;
            this.Birthdate = birthdate;
        }
        public string Name { get; set; }

        public int Age { get; set; }

        public string Id { get; set; }
        public string Birthdate { get; set; }
    }
}

File: SoftUni - Java OOP/Interfaces and Abstraction/Interfaces and Abstraction - Exercises/_02_Multiple_Implementation/Citizen.java
package _02_Multiple_Implementation;

public class Citizen implements Person, Birthable, Identifiable{
    private String name;
    private int age;
    private String id;
    private String birthDate;
    public Citizen(String name, int age, String id, String birthDate) {
        setName(name);
        setAge(age);
        setId(id);
       setBirthDate(birthDate);

    }
    @Override
    public String getName() {
        return this.name;
    }

    @Override
    public int getAge() {
        return this.age;
    }

    private void setName(String name) {
        this.name = name;
    }

    private void setAge(int age) {
        this.age = age;
    }

    public String getId() {
        return id;
    }

    private void setId(String id) {
        this.id = id;
    }

    public String getBirthDate() {
        return birthDate;
    }

    private void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }
}
